go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/c2sp/wycheproof
+
+go 1.23.6
+
+require (
+	github.com/santhosh-tekuri/jsonschema v1.2.4 // indirect
+	github.com/santhosh-tekuri/jsonschema/v6 v6.0.1 // indirect
+	golang.org/x/text v0.14.0 // indirect
```

Comment:
Needs `go mod tidy` (to remove the `// indirect`).

Reply:
Done (and added a small CI step to verify we don't regress).